lib/travis/api/v3/queries/builds.rb

```diff
@@ -17,13 +17,15 @@ def active_from(repositories)
     end
 
     def filter(relation)
+      relation = relation.select(Build.column_names - ['config']) unless includes? 'build.config'.freeze
```

Comment:
We currently don't support `build.config`, so I think the `if` here could be dropped